src/App.vue

```diff
@@ -98,13 +98,158 @@
             </el-row>
         </div>
 
-        <el-row v-else>
+        <el-row v-else> <!-- SIDE BAR - INDEX -->
             <el-col :span="4">
                 <el-menu class="has-full-height" @select="changeTableActive">
+
+                    <div class="add-delete-buttons">
+                        <el-button
+                            plain
+                            title="Create database"
+                            class="initial-buttons"
+                            @click="toggleModal('New database', true, 'database')"
+                        >
+                            <i class="first-icon fa fa-plus-circle"></i>
+                            <i class="second-icon fa fa-database"></i>
+                        </el-button>
+
+                        <el-button
+                            plain
+                            title="Remove database"
+                            class="initial-buttons"
+                            @click="toggleModal('Delete database', false, 'database')"
+                        >
+                            <i class="first-icon fa fa-minus-circle"></i>
+                            <i class="second-icon fa fa-database"></i>
+                        </el-button>
+
+                        <el-button
+                            plain
+                            title="Create table"
+                            class="initial-buttons"
+                            @click="toggleModal('New table', true, 'table')"
+                        >
+                            <i class="first-icon fa fa-plus-circle"></i>
+                            <i class="second-icon fa fa-table"></i>
+                        </el-button>
+
+                        <el-button
+                            plain
+                            title="Remove table"
+                            class="initial-buttons"
+                            @click="toggleModal('Delete table', false, 'table')"
+                        >
+                            <i class="first-icon fa fa-minus-circle"></i>
+                            <i class="second-icon fa fa-table"></i>
+                        </el-button>
+                    </div>
+
+
+                    <el-dialog
+                        :visible.sync="showDialogCreate"
+                        :title="dialogTitle"
+                        :close-on-click-modal="false"
+                        @click="closeModal()"
+                    >
+                        <div class="form_holder">
+                            <div v-if="showAddInput">
```

Comment:
The extra div here is on purpose? because you can use the `<template>` tag, it will not generate any extra DOM